src/pudl/metadata/resources/eiaaeo.py

```diff
@@ -0,0 +1,524 @@
+            ],
+            "primary_key": [
+                "report_year",
+                "electricity_market_module_region_aeo",
+                "modeling_case_aeo",
+                "projection_year",
+            ],
+        },
+        "sources": ["eiaaeo"],
+        "field_namespace": "eiaaeo",
+        "etl_group": "eiaaeo",
+    },
+    "core_eiaaeo__yearly_projection_end_use_sectors_generation": {
+        "description": (
+            "Projected total generation in the end-use sector, across all "
+            "fuel types.\n"
+            "Includes combined-heat-and-power plants and electricity-only "
+            "plants in the commercial and industrial sectors; and small on-site "
+            "generating systems in the residential, commercial, and industrial "
+            "sectors used primarily for own-use generation, but which may also "
+            "sell some power to the grid. Solar photovoltaic capacity portion of "
+            "Renewable Sources in gigawatts direct current; other technologies "
+            "in gigawatts alternating current."
+        ),
+        "schema": {
+            "fields": [
+                "report_year",
+                "electricity_market_module_region_aeo",
+                "modeling_case_aeo",
+                "projection_year",
+                "gross_generation_mwh",
+                "sales_mwh",
+                "generation_for_own_use_mwh",
+            ],
+            "primary_key": [
+                "report_year",
+                "electricity_market_module_region_aeo",
+                "modeling_case_aeo",
+                "projection_year",
+            ],
+        },
+        "sources": ["eiaaeo"],
+        "field_namespace": "eiaaeo",
+        "etl_group": "eiaaeo",
+    },
+    "core_eiaaeo__yearly_projection_electricity_sales": {
+        "description": "Projected electricity sales.",
+        "schema": {
+            "fields": [
+                "report_year",
+                "electricity_market_module_region_aeo",
+                "modeling_case_aeo",
+                "projection_year",
+                "customer_class",
+                "sales_mwh",
+            ],
+            "primary_key": [
+                "report_year",
+                "electricity_market_module_region_aeo",
+                "modeling_case_aeo",
+                "projection_year",
+                "customer_class",
+            ],
+        },
+        "sources": ["eiaaeo"],
+        "field_namespace": "eiaaeo",
+        "etl_group": "eiaaeo",
+    },
+    "core_eiaaeo__yearly_projection_net_energy_for_load": {
+        "description": "Projected net energy for load.",
+        "schema": {
+            "fields": [
+                "report_year",
+                "electricity_market_module_region_aeo",
+                "modeling_case_aeo",
+                "projection_year",
+                "generation_source_or_sink_type_aeo",
+                "net_energy_for_load_mwh",
```

Comment:
could this guy be `net_generation_for_load_mwh`? that would be more consistent w/ our other net gen names. alternatively we do have a `net_load_mwh` field which looks applicable

Reply:
ah yes i see. I think i would be partial to using the existing field `net_load_mwh`. I think right now it shows up in the context of pumped hydro plants in ferc1 tables but that seems relevant in a source/sink context. but this is not a hill i will die on i think what you have here is fine.

Reply:
I guess the table name also provides context. Yeah, let's go for `net_load_mwh`.